datasets/doc/source/tutorial-quickstart.rst

```diff
@@ -5,7 +5,7 @@ Run Flower Datasets as fast as possible by learning only the essentials.
 
 Install Federated Datasets
 --------------------------
-Run on the command line
+On the command line, run
 
 .. code-block:: bash
```

Comment:
Should we indicate that:
```
python -m pip install flwr-datasets[vision]
```
might not work depending on the shell you use? for instnace for me on `zsh` using `[` never works. I need to scape them:
```
python -m pip install flwr-datasets\[vision\]
```

Maybe worth having a comment like: "depending on your shell, you might need to wrap `[vision]` with `\[vision\]`

Reply:
Does it work if you make it a string? so `"flwr-datasets[vision]"`

Reply:
yes, it works

Reply:
So, let me change it to a string.